object-search/pods/vec.yml

```diff
@@ -18,6 +18,7 @@ metas:
 requests:
   on:
     IndexRequest:
+      # - !Buffer2URI {}
```

Comment:
Ok try putting between `!VectorIndexDriver` and` !KVIndexDriver` a `!Blob2PngURI` driver. I Think that is the buggest difference with cxmodal search, is that at this point, u are indexing the ouptut of a segmenter that provides ''blob'' and not 'buffer' directly in output